### PR TITLE
fix/can redirect to login

### PR DESCRIPTION
**Pull request by prankur34**
### Issue :

- the user should not be redirected to the fake page when clicking on '**login with github** ' button , instead if GitHub is logged in the user should be redirected to the RDS homepage , if not logged in should be redirected to github login.

### Files Changed 

- index.js 

### Fixes :
The user will not be redirected to the fake page , will be redirected to the real github login page .

**For better understanding and more reference, Please read the issue ticket:** #301 

### Before fix video : 
https://www.loom.com/share/7b6d9439f31e4a43b07f7f5c3faff82c

### After fix video :
https://www.loom.com/share/31a39f610bb647fb9b6b1a6601fc2d89

### CODE REVIEW

**Comment by RitikJaiswal75** [comment]
- Please write a proper description of the PR.
- include the keyword `closes: #issue_Number` to automatically close the issue ticket.

File: js/github.js
Comment (ankushdharkar) [inline js/github.js:15]: This doesn't look like the right place for making the override.

Could you please take care of this, where the switch between the real button and fake button takes place?

**Review by ankushdharkar — CHANGES_REQUESTED** [review]

File: js/github.js
Comment (prankur34) [reply js/github.js:15]: Sure will fix it .

**Comment by RitikJaiswal75** [comment]
Use the keyword `closes` or attach the issue under the development so the issue auto-closes. Also where is the video of the solution?

File: js/index.js
Comment (RitikJaiswal75) [inline js/index.js:76]: Please don't keep this unused code

No need to comment it out just remove it

**Review by RitikJaiswal75 — CHANGES_REQUESTED** [review]

File: js/index.js
Comment (prankur34) [reply js/index.js]: Hey @RitikJaiswal75  I have removed the commented code please review and approve if its fine .

File: js/github.js
Comment (prankur34) [reply js/github.js]: I have updated the code and removed the conditional button switching part, if it's fine; please approve the PR. @ankushdharkar

**Comment by ivinayakg** [comment]
# DO NOT MERGE
There is a PR of mine that integrates the new GitHub flow and only the prior GitHub mock has been removed.
 - PR -> https://github.com/Real-Dev-Squad/website-www/pull/277

**Comment by prankur34** [comment]
Thanks for the update .  @ankushdharkar

File: js/index.js
Comment (rohan09-raj) [inline js/index.js:73]: Can you please remove the import function also from `github.js`

**Comment by RitikJaiswal75** [comment]
@prankur34 Please close this PR

**Comment by RitikJaiswal75** [comment]
The issue is solved by @ivinayakg 
So closing this PR.

**Pull request closed without merging**
